### PR TITLE
Add a Log handler, to emit metrics via logger; Vagrant setup.

### PR DESCRIPTION
**Pull request by mikepea**
This adds a Log handler that can be used to test metrics collection
without actually sending to a remote service: metrics are outputted as
JSON via the logrus logger. 

This also adds a Vagrant config, for local development and testing.

### CODE REVIEW

File: src/fullerite/handler/log.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	h.configureCommonParams(configMap)
+}
+
+// Run runs the handler main loop
+func (h *Log) Run() {
+	h.run(h.emitMetrics)
+}
+
+func (h *Log) convertToLog(incomingMetric metric.Metric) string {
+	jsonOut, _ := json.Marshal(incomingMetric)
```
Comment (tsheasha) [inline src/fullerite/handler/log.go]: handle errors while marshalling?

Comment (mikepea) [inline src/fullerite/handler/log.go]: ooh, good spot. /me remembers to do a `git grep ' _ '` before pushing :)

**Comment by gnufied** [comment]
looks pretty good to me. only thing is - hopefully `Log` name does not cause too many clashes as the code grows, because it is defined and exported.

**Comment by tsheasha** [comment]
@mikepea +1 fix and ship

**Comment by mikepea** [comment]
@tsheasha all done, i'll let you take the call to merge.

**Comment by tsheasha** [comment]
@mikepea lgtm!

**Pull request merged**